Cargo.toml

```diff
@@ -47,7 +47,10 @@ members = [
 ]
 
 [patch.crates-io]
-ethereum-types = { path = "./patches/ethereum-types-0.10.0-to-0.11.0" }
+# Temporary wasmer fork which disables rayon (parallel compilation)
+# https://github.com/wasmerio/wasmer/pull/2262
+# Revome this when the new version of wasmer is published.
```

Comment:
```suggestion
# Remove this when the new version of wasmer is published.
```

Reply:
This lines and comments are dropped in a later commit (new version of wasmer shipped with feature to disable rayon) Thanks!